components/create-bounty-token-amount.tsx

```diff
@@ -0,0 +1,134 @@
+  defaultToken,
+  userAddress,
+  customTokens,
+  labelSelect,
+  tokenBalance,
+  issueAmount,
+  setIssueAmount,
+  review = false,
+  isFundingType,
+}) {
+  const { t } = useTranslation("bounty");
+
+  function getCurrentCoin() {
+    return customTokens?.find((token) => token?.address === currentToken);
+  }
+
+  function handleIssueAmountOnValueChange(values: NumberFormatValues) {
+    if (values.floatValue < 0 || values.value === "-") {
+      setIssueAmount({ formattedValue: "" })
+    } else {
+      setIssueAmount(values)
+    }
+  }
+
+  function handleIssueAmountBlurChange() {
+    if (isFundingType  && issueAmount.floatValue > tokenBalance) {
+      setIssueAmount({ formattedValue: tokenBalance.toString() });
+    }
+  }
+
+  function handleHelperText() {
+    if (review) return;
+    if (!isFundingType) return;
+    return (
+      <>
+        {t("fields.amount.info", {
+          token: currentToken?.symbol,
+          amount: formatNumberToCurrency(tokenBalance, {
+            maximumFractionDigits: 18,
+          }),
+        })}
+        <span
+          className="caption-small text-primary ml-1 cursor-pointer text-uppercase"
+          onClick={() =>
+            setIssueAmount({
+              formattedValue: tokenBalance.toString(),
+            })
+          }
+        >
+          {t("fields.amount.max")}
+        </span>
+      </>
+    );
+  }
+
+  return (
+    <div className="container">
+      <div className="col-md-12 mt-4">
+        <TokensDropdown
+          label={labelSelect}
+          tokens={customTokens}
+          userAddress={userAddress}
+          defaultToken={defaultToken}
+          canAddToken={canAddCustomToken}
+          addToken={addToken}
+          setToken={setCurrentToken}
+          disabled={review}
+        />
+      </div>
+      <div className="col-md-12">
+        <div className="d-flex">
+          <InputNumber
+            thousandSeparator
+            disabled={review}
+            max={tokenBalance}
+            label={t("fields.amount.label", {
+              token: currentToken?.symbol,
+            })}
+            symbol={currentToken?.symbol}
+            value={issueAmount.formattedValue}
+            placeholder="0"
+            onValueChange={handleIssueAmountOnValueChange}
+            onBlur={handleIssueAmountBlurChange}
+            helperText={handleHelperText()}
+          />
+          <div className="mt-4 pt-1 mx-2">
+            <ArrowRight className="text-gray" width={9} height={9} />
+          </div>
+          <InputNumber
+            thousandSeparator
+            label={" "}
+            className="mt-3"
+            symbol={"EUR"}
+            classSymbol="text-white-30 mt-3"
+            disabled={true}
+            value={
+              getCurrentCoin()?.tokenInfo
+                ? handleTokenToEurConversion(Number(issueAmount.value),
+                                             getCurrentCoin()?.tokenInfo?.prices["eur"])
+                : "-"
```

Comment:
this value makes no sense. Either remove it (and the placeholder takes hold) or put it at 0

Reply:
This condition is used when the coingecko api does not return information. if you put 0, it won't give the idea that this coin is worth 0 euros, right?